examples/grpc_package_example/Package.swift

```diff
@@ -3,9 +3,11 @@
 import PackageDescription
 
 let package = Package(
-    name: "grpc_package_example",
+    name: "grpc_example",
```

Comment:
Curious. Why did you rename this? The original name seems better to me.

Reply:
Ah I copy-pasted the Package.swift file from the other example because the required packages were the same. I can revert the package name.